src/main/java/edu/hw1/Task3.java

```diff
@@ -0,0 +1,58 @@
+package edu.hw1;
+
+public final class Task3 {
+    private Task3() {
+    }
+
+    /**
+     * Возвращает минимум непустого массива
+     * @param array массив, минимум которого нужно найти
+     * @return минимум массива array
+     */
+    private static int getMin(int[] array) {
+        int ret = array[0];
```

Comment:
`bug` дай нормальное назваине
```suggestion
        int currentMin = array[0];
```